tests/test_integration.py

```diff
@@ -2294,13 +2344,28 @@ async def test_statistics_json(envsonschema, always_valid):
         json=True,
     )
 
+    last_ran_on_str = jsonout["last_ran_on"]
+    assert last_ran_on_str is not None, "Last ran on date not found in jsonout"
```

Comment:
This doesn't seem like it will ever be None, if it's missing it'll have raised an exception and errored the test (which is fine).